src/main/kotlin/com/exactpro/th2/lwdataprovider/db/CradleMessageExtractor.kt

```diff
@@ -122,7 +127,17 @@ class CradleMessageExtractor(configuration: Configuration, private val cradleMan
         val id = storedMessage.id.toString()
         val decodingStep = startStep("decoding")
         val tmp = createMessageDetails(id, 0, storedMessage) { decodingStep.finish() }
-        tmp.rawMessage = startStep("raw_message_parsing").use { RawMessage.parseFrom(storedMessage.content) }.also {
+        tmp.rawMessage = startStep("raw_message_parsing").use {
+            RawMessage.newBuilder().apply {
```

Comment:
Looks like there are several places with the same logic. Probably, we should extract it into a method and reuse